src/Files.App/Data/Items/ShellLibraryItem.cs

```diff
@@ -9,7 +9,7 @@ public sealed class ShellLibraryItem
 	{
 		public const string EXTENSION = ".library-ms";
 
-		public static readonly string LibrariesPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Microsoft", "Windows", "Libraries");
+		public static readonly string LibrariesPath = Win32PInvoke.GetFolderFromKnownFolderGUID(new Guid("1B3EA5DC-B587-4786-B4EF-BD1DC332AEAE"));
```

Comment:
Please create a new class named ‘CLSID’ (I was going to do so).
It enables us to add also other than special folders such as Com Interfaces and more.

Reply:
Under Win32PInovke?

Reply:
Under Files.App.Constants

Reply:
I'll leave this for you in a follow up PR.

Reply:
Ok...